db/models.py

```diff
@@ -1 +1,58 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return f"{self.name}. {self.description}"
+
+
+class Skill(models.Model):
+    name = models.CharField(unique=True, max_length=255)
+    bonus = models.CharField(
+        "Describes what kind of bonus players can get from it",
+        max_length=255
+    )
+    race = models.ForeignKey(
+        "Race",
+        related_name="skill_set",
+        on_delete=models.DO_NOTHING
```

Comment:
I believe skills can’t live without race so you need to change the deleting type
Also thy to avoid `DO_NOTHING`